README.md

```diff
@@ -52,7 +52,11 @@ Live Community Chat: [![Gitter](https://img.shields.io/badge/gitter-join_chat-bl
 
 ### Building NuPIC From Source Code
```

Comment:
Perhaps change this to `Installing NuPIC from Source`? There isn't really a build for this pure-Python project.

Also, we might want to mention that if you want to build nupic.bindings from source that you should build and install from nupic.core prior to installing nupic (since a PyPI release will be installed if nupic.bindings isn't yet installed).